en/includes/guides/organization-management/onboard-users.md

```diff
@@ -0,0 +1,158 @@
+# Onboard users to organizations
+
+Organization administrators can add users to their organization using any of the following methods.
+
+!!! note "Before you begin"
+
+    - Make sure you have created an organization in {{product_name}} and onboarded an administrator. See how to [create an organization]({{base_path}}/guides/organization-management/) and [onboard admins]({{base_path}}/guides/organization-management/onboard-users/).
+    - In the following guides, the organization's management interface is referred to as the Console. Make sure you switch to the correct organization and perform actions within its Console. Learn how to [switch organizations]({{base_path}}/guides/organization-management/manage-organizations/#switch-to-an-organization).
+
+## Onboard new users from the Console
+
+Organization administrators can use the Console to create users in the organization. Users can be added individually or as a collection from the Console.
+
+- To add a single user, follow this [guide]({{base_path}}/guides/users/manage-users/#onboard-single-user).
+- To add multiple users, follow this [guide]({{base_path}}/guides/users/manage-users/#onboard-multiple-users).
+
+## Onboard users from a secondary user store
```

Comment:
Shall we restrict this only to IS 7.1 and onwards since the feature is added to 7.1?

Reply:
fixed with https://github.com/wso2/docs-is/pull/4981/commits/4c4def4725190873f5977d899f367047502b3d2a